### PR TITLE
Feature/add zaakdetails

### PR DESCRIPTION
**Pull request by flinden68**

### CODE REVIEW

File: app/src/main/resources/config/application.yml [hunk 1]
```diff
@@ -67,6 +67,7 @@ valtimo:
             url: http://localhost:8001
             clientId: valtimo_client
             secret: e09b8bc5-5831-4618-ab28-41411304309d
+            detailsTypeUrl: "http://host.docker.internal:8011/api/v1/objecttypes/3e852115-277a-4570-873a-9a64be3aeb34"
```
Comment (theo-ritense) [inline app/src/main/resources/config/application.yml:70]: I feel like this property should be named `objectTypeUrl`.

File: zgw/zaken-api/src/main/kotlin/com/ritense/portal/zakenapi/domain/ZaakDetailsObject.kt
Comment (theo-ritense) [inline zgw/zaken-api/src/main/kotlin/com/ritense/portal/zakenapi/domain/ZaakDetailsObject.kt:1]: This class is identical to `ZaakDetails`. Does it have any special purpose? If so, please try to find a name that describes that purpose. If not, please remove it and use `ZaakDetails` instead.

File: zgw/zaken-api/src/test/resources/config/application.yml [hunk 2]
```diff
@@ -20,3 +20,6 @@ valtimo:
             url: #filled in test
             clientId: valtimo_client
             secret: e09b8bc5-5831-4618-ab28-41411304309d
+            detailsTypeUrl: "http://host.docker.internal:8011/api/v1/objecttypes/3e852115-277a-4570-873a-9a64be3aeb34"
```
Comment (theo-ritense) [inline zgw/zaken-api/src/test/resources/config/application.yml:23]: I feel like this property should be named objectTypeUrl.

File: app/src/main/resources/config/application.yml (hunk 1 above)
Comment (flinden68) [reply app/src/main/resources/config/application.yml:70]: Removed, not used

File: zgw/zaken-api/src/test/resources/config/application.yml (hunk 2 above)
Comment (flinden68) [reply zgw/zaken-api/src/test/resources/config/application.yml:23]: Removed, not used

File: zgw/zaken-api/src/main/kotlin/com/ritense/portal/zakenapi/domain/ZaakDetailsObject.kt
Comment (flinden68) [reply zgw/zaken-api/src/main/kotlin/com/ritense/portal/zakenapi/domain/ZaakDetailsObject.kt:1]: ZaakDetailsObject is the zaakdetails object from the objectsAPI, based on a list of ZaakObjects. In the code it loops through the zaakObjects and collect from ZaakDetailsObject the data and store it in the data of the data list of the ZaakDetails. There could be more.
On different locations, like in the taak module the Taak Object from the ObjectsAPI also has the name of TaakObject, so I used the same principle

File: app/src/main/resources/config/application.yml (hunk 1 above)
Comment (theo-ritense) [reply app/src/main/resources/config/application.yml:70]: I think you forgot to push the change, as I can still see it in the PR.

**Review by theo-ritense — APPROVED** [review]

**Pull request merged**